tools/depends/configure.ac

```diff
@@ -325,7 +332,7 @@ case $host in
           if test "x$use_cpu" = "xauto"; then
             use_cpu=arm64
           fi
-
+          use_cmake_cpu=aarch64
```

Comment:
Here a small difference to Kodi Matrix, it have `if test "x$use_cpu" = "xauto"; then` before on Matrix not.